File: Python/problem1672.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        # import sys
        # max_ = -sys.maxsize
        # for account in accounts:
        #     if sum(account) > max_:
        #         max_ = sum(account)
        # return max_

        # accounts.sort(key=lambda x: sum(x), reverse=True)
        # return sum(accounts[0])

        return sum(sorted(accounts, key=lambda x: sum(x), reverse=True)[0])

File: C++/problem1672.cpp
#include <numeric>

class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int max_ = 0;
        for (vector<vector<int>>::iterator it = accounts.begin(); it < accounts.end(); ++it)
        {
            int tmp = accumulate((*it).begin(), (*it).end(), 0);
            if (tmp > max_)
            {
                max_ = tmp;
            }
        }
        return max_;
    }
};
